make/Images.gmk

```diff
@@ -88,6 +88,16 @@ ifeq ($(JLINK_KEEP_PACKAGED_MODULES), true)
   JLINK_JDK_EXTRA_OPTS := --keep-packaged-modules $(JDK_IMAGE_DIR)/jmods
 endif
 
+ifeq ($(BUILD_CDS_ARCHIVE), true)
+  ifeq ($(OPENJDK_TARGET_OS), windows)
+		CDS_ARCHIVE := bin/server/classes.jsa
+		CDS_NOCOOPS_ARCHIVE := bin/server/classes_nocoops.jsa
```

Comment:
Where/ how is this variable used? It seems as though lines 94 and 97 are the only mentions of this variable in the changed lines

Reply:
Removed!